### PR TITLE
map update

### PR DESCRIPTION
**Pull request by LetterN**
## About The Pull Request
Puts some of the latest tg map things in here

## Changelog
:cl:
add: Box Station: Nanites Lab
tweak: Box Station: R&D Lab is now bigger
tweak: Box Station: Resizes the testing lab, circuit lab, and toxins
add: Meta Station: Nanites lab
add: Meta Station: Engineering shared storage (includes the engi protolathe)
add: Pubby Station: Engineering shared storage
tweak: Pubby Station: Expanded Medbay lobby
del: Pubby Station: removes the extra central hallway
/:cl:

### CODE REVIEW

**Comment by KathrinBailey** [comment]
No pics?

**Comment by LetterN** [comment]
check mapdiffbot

**Review by Ghommie — CHANGES_REQUESTED** [review]
A disposal bin blocks the entrace to Boxstation experimentor room. Also its experimentor machine now hugging the wall seems a tad annoying, at least to me.
There seems to be a rogue floor decal on the new Metastation engineering room, by the table, near the APC, beside the engi vendors.
Pubby's medical seems to have the most problems: A medikit casually lying on the floor and a directional free acess (?) airlock helper one tile right the CMO's office entrace. Both of these tiles now also happen to be missing their former decals. There is an airlock beside the sleepers corner awkwardly facing, again, a disposal bin, and another, maintenance, airlock south of the surgery smartfridge. Also the crew med sensors console new location is very uncomfortable, by a corner betwixt two airlocks, really...

Hopefully it's not the map merger acting up.

EDIT: Did I say map merger? I meant diff bot.

**Comment by LetterN** [comment]
> A disposal bin blocks the entrace to Boxstation experimentor room. Also its experimentor machine now hugging the wall seems a tad annoying, at least to me.

will fix that
>There seems to be a rogue floor decal on the new Metastation engineering room, by the table, near the APC, beside the engi vendors.

map render fault
>A medikit casually lying on the floor and a directional free acess (?) airlock helper one tile right the CMO's office entrace. Both of these tiles now also happen to be missing their former decals. There is an airlock beside the sleepers corner awkwardly facing, again, a disposal bin, and another, maintenance, airlock south of the surgery smartfridge.

not a feature, will fix

**Comment by KathrinBailey** [comment]
There's also a breach in Meta atmos with a plating plus RC in place of glass.
One of the markers for the engineering escape pod is missing on Meta.
Also might be worth making the tile in front of the engi circuit imprinter on Meta into rglass so its a little more consistent and there's increased visibility into the lathe room.

File: _maps/map_files/BoxStation/BoxStation.dmm [hunk 1]
```diff
@@ -32502,21 +32257,18 @@
...
 /obj/structure/cable{
 	icon_state = "1-2"
 	},
 /turf/open/floor/plasteel/white,
 /area/science/research)
 "bBF" = (
-/obj/machinery/portable_atmospherics/scrubber,
-/obj/item/storage/firstaid/toxin,
-/obj/effect/turf_decal/stripes/line,
-/obj/machinery/airalarm/unlocked{
```
Comment (Trilbyspaceclone) [inline _maps/map_files/BoxStation/BoxStation.dmm:32515]: Please dont replace this with an locked airalarm

**Comment by Trilbyspaceclone** [comment]
I dont like the box changes at all tbh...

**Comment by Ghommie** [comment]
I do like the changes. Also merge conflicts.

**Comment by Ghommie** [comment]
Travis reports an unidentified map symbol on Meta. Uh uh.

**Comment by LetterN** [comment]
what the fuck travis, this isn't causing an error on my side

File: _maps/map_files/BoxStation/BoxStation.dmm (hunk 1 above)
Comment (LetterN) [reply _maps/map_files/BoxStation/BoxStation.dmm:32515]: it's now unlocked in meta

**Review by deathride58 — CHANGES_REQUESTED** [review]
Changes to the sec lobby on box were regressed. The randomized map engines were regressed. Extra lights in box's xenobio cells were removed. Roundstart nanite labs are undocumented (requires further discussion with maintainers). Fixes to the command area on box were regressed. Grav gen is suddenly missing lights and probably power as well. The disabler shotgun removal was regressed.

This is just boxstation alone. Please document your changes.

**Review by Ghommie — APPROVED** [review]
Looks fine, a few peeves may still linger around but it looks fine for testmerging.

**Comment by Ghommie** [comment]
EOL conflicts, otherwise mergeable.

**Comment by ghost** [comment]
the mapdiff bot has been checking this for over 2 days now

**Comment by Ghommie** [comment]
sadly, more conflicts.

**Pull request closed without merging**